templates/dynamodb.yml

```diff
@@ -2,7 +2,6 @@ Resources:
   UserTable:
     Type: AWS::DynamoDB::Table
     Properties:
-      TableName: ${{self:provider.stage}}-users
       BillingMode: PAY_PER_REQUEST
       PointInTimeRecoverySpecification:
```

Comment:
@bboure anything else you can think of we'll need to do to get non-hardcoded names working?

Reply:
@brettstack no, it should keep working like [this](https://github.com/RareCamp/RareCamp/blob/master/templates/graphql.yml#L49-L51)